website/views/shared/avatar/index.jade

```diff
@@ -26,5 +26,7 @@ mixin herobox(opts)
       span.glyphicon.glyphicon-circle-arrow-up(tooltip=env.t('buffed'), tooltip-append-to-body="true",
         ng-show='#{buffs}.str || #{buffs}.per || #{buffs}.con || #{buffs}.int || #{buffs}.stealth')
       span(tooltip=env.t('level'), tooltip-append-to-body="true") {{profile.stats.lvl}}
-      span.glyphicon.glyphicon-plus-sign(ng-show='profile.achievements.rebirths',
+      span.glyphicon.glyphicon-plus-sign(ng-if='::profile.achievements.rebirthLevel < 100', ng-show='profile.achievements.rebirths',
         tooltip=env.t('reborn', {reLevel: "{{profile.achievements.rebirthLevel}}"}), tooltip-append-to-body="true")
```

Comment:
I feel like you could do something like:

```jade
- var rebirthLevel = "{{profile.achievements.rebirthLevel < 100 ? profile.achievements.rebirthLevel : 100+}}"
span.glyphicon.glyphicon-plus-sign(ng-show='profile.achievements.rebirths',
  tooltip=env.t('reborn', {reLevel: rebirthLevel}), tooltip-append-to-body="true")
```

That would eliminate the need for an extra translation string.